packages/tempus/src/Tempus_StepperDIRK_decl.hpp

```diff
@@ -234,6 +234,8 @@ class StepperDIRK : virtual public Tempus::StepperImplicit<Scalar>,
     virtual void setUseEmbedded(bool a) { useEmbedded_ = a; }
     virtual bool getUseEmbedded() const { return useEmbedded_; }
     virtual bool getUseEmbeddedDefault() const { return false; }
+    virtual void setTVDCoeff(const Scalar a) { tvdCoeff_ = a; }
+    virtual void setTVD(bool a) { isTVD_ = a; }
```

Comment:
Now that we have StepperRKBase, these set functions might be better there instead of duplicating them here and ExplicitRK (and IMEX?).

Reply:
ok